helpers/manifest.json

```diff
@@ -924,5 +924,34 @@
         "max": "50"
       }
     ]
+  },
+  { 
+    "name": "Hover indicator",
+    "author": "minnieo",
+    "version": "0.1.0",
+    "label": "Hover indicator",
+    "login": false,
+    "recurs": false,
+    "desc": "Apply an outline to elements that are being hovered over.",
+    "namespace": "hover",
+    "entrypoint": "hoverIndicator",
+    "page": "accessibility",
+    "fields": [
+      {
+        "type": "color",
+        "initial": "#FFFFFF",
```

Comment:
Same question as the other script: can we start this at the default hover color (e.g. `--kbin-vote-text-hover-color`)? Feels like `FFFFFF` might not render on the white theme.